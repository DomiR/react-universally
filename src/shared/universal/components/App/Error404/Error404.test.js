/* eslint-disable no-undef */
/* eslint-disable import/no-extraneous-dependencies */
import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Error404 from './Error404';

describe('<Error404 />', () => {
  test('Renders the component', () => {
    const wrapper = shallow(<Error404 />);
    expect(wrapper.find('div').length).toBe(1);
  });
  test('renders a snapshot', () => {
    const wrapper = shallow(<Error404 />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
