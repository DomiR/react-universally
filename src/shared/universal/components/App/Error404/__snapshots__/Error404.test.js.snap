exports[`<Error404 /> renders a snapshot 1`] = `
<div>
  Sorry, that page was not found.
</div>
`;
